aos-vm.yaml

```diff
@@ -311,6 +311,7 @@ parameters:
             builder:
               conf:
                 - [AOS_MAIN_NODE, "1"]
+                - [AOS_INCLUDE_MESSAGEPROXY, "0"]
```

Comment:
Should be another approach without modifying meta-aos, namely:

```yaml
parameters:
  ...
  WITH_MESSAGE_PROXY:
    desc: "Enable Aos message proxy"
    "yes":
      overrides:
        components:
          aos-vm:
            builder:
              conf:
                - [
                    IMAGE_INSTALL:append,
                    "  ${@'aos-messageproxy' if bb.utils.to_boolean(d.getVar('AOS_MAIN_NODE')) else ''}",
                  ]

    "no":
      default: true
```

Reply:
applied